### PR TITLE
EASY-2689: Make SHA-1 manifest mandatory for AIP only

### PR DESCRIPTION
**Pull request by janvanmansum**
Fixes EASY-2869

#### When applied it will
* Make rule about SHA-1 being mandatory AIP-only.

#### Related PR
* [x] https://github.com/DANS-KNAW/easy-validate-dans-bag/pull/82
* [x] awaiting PR on `easy-ingest-flow` to be completed (EASY-2690, https://github.com/DANS-KNAW/easy-ingest-flow/pull/141)

### CODE REVIEW

File: versions/0.0.0.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ Requirements
    its value SHOULD be the user name of an existing EASY user account.
 
 #### 1.3 Manifests
-1. (a) The bag MUST have a SHA-1 payload manifest. (b) It MUST have entries for all the payload files.
+1. **(AIP)** (a) The bag MUST have a SHA-1 payload manifest. (b) It MUST have entries for all the payload files.
```
Comment (lindareijnhoudt) [inline versions/0.0.0.md]:
```suggestion
1. **(SIP)** (a) The bag MUST have a payload manifest. (b) It MUST have entries for all the payload files.
**(AIP)** (a) The bag MUST have a SHA-1 payload manifest. (b) It MUST have entries for all the payload files.
```

Comment (lindareijnhoudt) [reply versions/0.0.0.md]: SIPs also need a payload manifest. Should we enumerate, or give examples?

Comment (janvanmansum) [reply versions/0.0.0.md]: The requirement that there is at least *some* payload manifest is covered by the requirement that the bag is valid.

Comment (janvanmansum) [reply versions/0.0.0.md]: We *could* consider requiring a minimum strength of SHA-1 for the SIP payloadmanifest.

**Pull request merged**